EthOn.rdf

```diff
@@ -1904,18 +1904,18 @@
         </owl:annotatedTarget>
         <rdfs:comment>All create transactions are directed toward a Contract Account they create.</rdfs:comment>
     </owl:Axiom>
-
+    
 
 
     <!-- http://ethon.consensys.net/EthOnConcept -->
 
     <owl:Class rdf:about="http://ethon.consensys.net/EthOnConcept">
         <suggestedStringRepresentation xml:lang="en">EthonConcept</suggestedStringRepresentation>
-        <rdfs:comment xml:lang="en">Groups al EthOn concepts.</rdfs:comment>
+        <rdfs:comment xml:lang="en">The purpose of concepts in EthOn is to group a finite number of concrete instances of states and/or relationships between states under a single indefinitely bounded term.</rdfs:comment>
```

Comment:
@jo-tud OK no problem! Maybe we could change it thus:


This is the foundational concept of all concepts that are original to the EthOn OWL ontology.

To me, root indicates an organic growth from a central principle whereas foundation indicates a more synthetic/representation-oriented mode of grouping and organizing, since Ethereum grows organically but EthOn does not as EthOn's goal is to represent systematically what already exists in Ethereum, for EthOn concept I would go for "foundational" over "root."